main.js

```diff
@@ -0,0 +1,155 @@
+// Functions for player vs player mode \\
+
+player2.addEventListener('click', () => {
+    startTimer()
+    event.target.disabled = true;
+    status.textContent = 'Player X, your turn!'
+})
+let counter = 0
+twoPlayer()
+function twoPlayer() {
+    for (let cell of board) {
+        cell.addEventListener('click', () => {
+            counter = counter + 1
+            if (counter === 9 && Winner === null) {
+                status.textContent = "Its a tie."
+                stopTimer()
+            }
+            if (cell.textContent === "X" || cell.textContent === "O") {
+                status.textContent = "Invalid move! Pick an empty spot."
+            }
+            if (status.textContent === 'Player X, your turn!') {
+                cell.textContent = 'X';
+                player = 'O'
+                status.textContent = 'Player O, your turn!'
+            } else if (status.textContent === 'Player O, your turn!') {
+                cell.textContent = 'O';
+                player = 'X'
+                status.textContent = 'Player X, your turn!'
+            }
+            if (cell1.textContent === 'X' && cell2.textContent === 'X' && cell3.textContent === 'X') { Winner = 'X' }
+            if (cell4.textContent === 'X' && cell5.textContent === 'X' && cell6.textContent === 'X') { Winner = 'X' }
+            if (cell7.textContent === 'X' && cell8.textContent === 'X' && cell9.textContent === 'X') { Winner = 'X' }
+            if (cell1.textContent === 'X' && cell4.textContent === 'X' && cell7.textContent === 'X') { Winner = 'X' }
+            if (cell2.textContent === 'X' && cell5.textContent === 'X' && cell8.textContent === 'X') { Winner = 'X' }
+            if (cell3.textContent === 'X' && cell6.textContent === 'X' && cell9.textContent === 'X') { Winner = 'X' }
+            if (cell1.textContent === 'X' && cell5.textContent === 'X' && cell9.textContent === 'X') { Winner = 'X' }
+            if (cell3.textContent === 'X' && cell5.textContent === 'X' && cell7.textContent === 'X') { Winner = 'X' }
+            if (Winner === 'X') {
+                status.textContent = "Player X wins the game.";
+                stopTimer()
+            }
+            // Check if O wins
+            if (cell1.textContent === 'O' && cell2.textContent === 'O' && cell3.textContent === 'O') { Winner = 'O' }
+            if (cell4.textContent === 'O' && cell5.textContent === 'O' && cell6.textContent === 'O') { Winner = 'O' }
+            if (cell7.textContent === 'O' && cell8.textContent === 'O' && cell9.textContent === 'O') { Winner = 'O' }
+            if (cell1.textContent === 'O' && cell4.textContent === 'O' && cell7.textContent === 'O') { Winner = 'O' }
+            if (cell2.textContent === 'O' && cell5.textContent === 'O' && cell8.textContent === 'O') { Winner = 'O' }
+            if (cell3.textContent === 'O' && cell6.textContent === 'O' && cell9.textContent === 'O') { Winner = 'O' }
```

Comment:
This is a lot of code just to have it print out who won the game, and there are some ways that you could simplify it a bit. While you're likely always going to need to specify combination of cells that determine a winner at least once, there might be a way to make some kind of a "win check" function, that could be called on if X _or_ O wins the match. This would at least save you from having to type it out twice.